R/dash.R

```diff
@@ -305,12 +305,38 @@ Dash <- R6::R6Class(
           
           # have to format the response body like this
           # https://github.com/plotly/dash/blob/064c811d/dash/dash.py#L562-L584
```

Comment:
whew that's an old link! Perhaps update it, split the comment to cover single and multi separately?

Also I notice up a few lines is the 204 response for `no_update` - it's probably not strictly necessary but the py side turns multi-outputs where *all* outputs have been canceled into 204's as well.